paywall/src/data-iframe/index.2.0.ts

```diff
@@ -2,14 +2,49 @@ import Postmate from 'postmate'
 import { Web3Service } from '@unlock-protocol/unlock-js'
 import { BlockchainReader } from './BlockchainReader'
 import { BlockchainDataStorable } from './BlockchainDataStorable'
-import configure from '../config'
+
+declare var __ENVIRONMENT_VARIABLES__: any
+
+// TODO FIND A WAY TO MOVE THIS IN A BETTER PLACE
+// NOTE: NEVER PUT ENV VARS HERE
+const constants: { [key: string]: any } = {
+  dev: {
+    unlockAddress: '0x885EF47c3439ADE0CB9b33a4D3c534C99964Db93',
+    blockTime: 3000,
+    requiredConfirmations: 6,
+    defaultNetwork: 1984,
+  },
+  test: {
+    unlockAddress: '0x885EF47c3439ADE0CB9b33a4D3c534C99964Db93',
+    blockTime: 3000,
+    requiredConfirmations: 6,
+    defaultNetwork: 1984,
+  },
+  staging: {
+    unlockAddress: '0xD8C88BE5e8EB88E38E6ff5cE186d764676012B0b',
+    blockTime: 8000,
+    requiredConfirmations: 12,
+    defaultNetwork: 4,
+  },
+  prod: {
+    unlockAddress: '0x3d5409CcE1d45233dE1D4eBDEe74b8E004abDD13',
+    blockTime: 8000,
+    requiredConfirmations: 12,
+    defaultNetwork: 1,
+  },
+}
+
+const config = {
+  ...__ENVIRONMENT_VARIABLES__,
+  ...constants[__ENVIRONMENT_VARIABLES__.unlockEnv],
+}
 
 const {
   readOnlyProvider,
   unlockAddress,
   blockTime,
   requiredConfirmations,
-} = configure()
```

Comment:
`configure()` has a dependency on `next`, which we don't have access to here. Luckily all the required info is in the environment variables we bundle.